tools/accuracy_checker/accuracy_checker/adapters/detection.py

```diff
@@ -909,3 +909,84 @@ def softmax(x):
             result.append(DetectionPrediction(identifier, labels, det_scores, x_mins, y_mins, x_maxs, y_maxs))
 
         return result
+
+
+class UltraLightweightFaceDetectionAdapter(Adapter):
+    """
+    Class for converting output of Ultra-Lightweight Face Detection models to DetectionPrediction representation
+    """
+    __provider__ = 'ultra_lightweight_face_detection'
+
+    @classmethod
+    def validate_config(cls, config, fetch_only=False, **kwargs):
+        return super().validate_config(
+            config, fetch_only=fetch_only, on_extra_argument=ConfigValidator.ERROR_ON_EXTRA_ARGUMENT
+        )
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'scores_out': StringField(description="Scores output layer name."),
+            'boxes_out': StringField(description="Boxes output layer name."),
+        })
+
+        return parameters
+
+    def configure(self):
+        self.scores_out = self.get_value_from_config('scores_out')
+        self.boxes_out = self.get_value_from_config('boxes_out')
+
+        # Set default values
+        self.prob_threshold = 0.7
+        self.iou_threshold = 0.5
+
```

Comment:
Possibly it makes sense to make these parameters configurable

Reply:
Done